api/cl_khr_command_buffer.asciidoc

```diff
@@ -110,6 +100,29 @@ following reasons:
   other extensions layered on top to take advantage of them to provide additional
   mutable functionality.
 
+==== Command Synchronization
+
+Device-side {cl_sync_point_khr_TYPE} synchronization-points can be used within
+command-buffers to define command dependencies. This allows the commands of a
+command-buffer to execute out-of-order on a single <<compatible, compatible>>
+command-queue. The command-buffer itself has no inherent in-order/out-of-order
+property, this ordering is inferred from the command-queue used on command
+recording. {clEnqueueCommandBufferKHR} submissions to an out-of-order queue
+have the same execution semantics are other operations enqueued to an
+out-of-order queue, such as {clEnqueueFillBuffer}. Where execution between
+enqueued operations may happen concurrently unless dependencies between the
+operations are expressed with events.
```

Comment:
I like the updated text, but I don't think the last sentence is a complete sentence.  How about:

```suggestion
recording. {clEnqueueCommandBufferKHR} submissions to an out-of-order queue
have the same execution semantics are other operations enqueued to an
out-of-order queue, such as {clEnqueueFillBuffer}, where execution between
enqueued operations may happen concurrently unless dependencies between the
operations are expressed with events.
```